letsdrill.rb

```diff
@@ -1,16 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    "A"
+  elsif integer >= 80
+    "B"
+  elsif integer >= 70
+    "C"
+  elsif integer >= 60
+    "D"
+  else
+    "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+   array.sort! { |x, y| x.length <=> y.length }
+   array[0]
 end
```

Comment:
The use of sort! here is dangerous to data passing into the function and could surprise an unwary person who called this method on their array. See if you can determine why and also consider refactoring this to make it more friendly to its arguments.

Check out Enumerable#sort_by and Enumerable#min_by, too. [Docs](http://ruby-doc.org/core-2.2.3/Enumerable.html)